jenkins-integration/jenkins-jobs/common/scripts/job-steps/020_install_pe.sh

```diff
@@ -21,9 +21,11 @@ bundle exec beaker \
         --log-level debug \
         --no-color \
         --tests \
-jenkins-jobs/common/scripts/ops-deployment/hack_hostname_into_etc_hosts.rb,\
-jenkins-jobs/common/scripts/ops-deployment/disable_firewall.rb,\
-beaker/install/pe/10_install_pe.rb
+beaker/install/shared/hack_hostname_into_etc_hosts.rb,\
+beaker/install/shared/disable_firewall.rb,\
+beaker/install/pe/10_install_pe.rb,\
+beaker/install/shared/configure_permissive_server_auth.rb,\
+beaker/install/pe/99_restart_server.rb
```

Comment:
Not directly related to changes in this PR, but we may want to use the beaker `bounce_service` function here inside of "99_restart_server.rb", which uses `puppet resource <service> ensure=stopped|running` under the hood, instead of `service pe-puppetserver restart`.  I've heard that some OSes - maybe SLES? - don't support `service <service name> restart` whereas I think the `puppet resource <service> ensure=stopped|running` approach works more broadly.